docs/book/src/tasks/experimental-features/cluster-class/write-clusterclass.md

```diff
@@ -549,6 +550,23 @@ spec:
             type: string
 ```
 
+Even if OpenAPI schema allows defining free form objects, e.g.
```

Comment:
Not sure if you want to make it clearer but even objects with defined properties allow additional undefined properties today.

This section might imply to some users that this only happens when you don't define properties.

Reply:
I consider this more a bug than a feature, I don't think we should advertise about it and rally on the open issue to get this sorted out asap